specification/summary-care-record.yaml

```diff
@@ -217,6 +217,77 @@ paths:
               example:
                 $ref: components/examples/DocumentReferenceBundle.json
 
+  /$setPermission:
+    post:
+      summary: Update patient's consent to View SCR
+      operationId: acs
+      tags:
+        - SCR
+      parameters:
+        - $ref: '#/components/parameters/BearerAuthorization'
+        - $ref: '#/components/parameters/CorrelationID'
+        - $ref: '#/components/parameters/RequestID'
+      requestBody:
+        required: true
+        content:
+          application/fhir+json:
+            schema:
+              $ref: components/schemas/Parameters.yaml
+            example:
+              $ref: components/examples/SetPermission.json
+      responses: 
+        "200": 
+          description: Success response
+        "400":
+          description: Bad request
+          content:
+            application/fhir+json:
+              schema:
+                $ref: components/schemas/OperationOutcome.yaml
+              example:
+                resourceType: OperationOutcome
+                issue:
+                  - severity: error
+                    code: invalid
+                    details:
+                      text: Invalid value - xyz in field 'valueCoding.code'
+
+  /AuditEvent:
+    post:
+      summary: Override patient's dissent to View SCR
+      operationId: alert
+      tags:
+        - SCR
+      parameters:
+        - $ref: '#/components/parameters/BearerAuthorization'
+        - $ref: '#/components/parameters/CorrelationID'
+        - $ref: '#/components/parameters/RequestID'
+        - $ref: '#/components/parameters/RoleID'
+      requestBody:
+        required: true
+        content:
+          application/fhir+json:
+            schema:
+              $ref: components/schemas/AuditEvent.yaml
+            example:
+              $ref: components/examples/AuditEvent.json
+      responses:
+        "200":
+          description: Success response
```

Comment:
Should we have an example for a successful response?

Reply:
No, successful response is just HTTP 200 code, no body.